R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,105 @@
+# importCNA.GISTIC2.R
+#
+#
+# Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#'@param fFHG A vector of local fully qualified paths of Firehose GISTIC2 CNA file(s).
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
```

Comment:
Yes. This really would be scope creep for the function. (And that's why we go through design -> writing tests -> implementing code in that order.)